packages/block-library/src/breadcrumbs/edit.js

```diff
@@ -147,17 +132,21 @@ export default function BreadcrumbEdit( {
 		// This is needed because when we are showing the template in post editor we
 		// want to show the real breadcrumbs if we have the post type.
 		( templateSlug && ! postType ) ||
-		( breadcrumbsType === 'postWithAncestors' &&
-			! isPostTypeHierarchical ) ||
-		( breadcrumbsType === 'postWithTerms' && ! hasTermsAssigned );
+		( ! _showTerms && ! isPostTypeHierarchical ) ||
+		( _showTerms && ! hasTermsAssigned );
 	if ( showPlaceholder ) {
-		const placeholderItems = [
-			showHomeLink && __( 'Home' ),
-			// For now if we are adding this in a template show a generic placeholder.
-			...( templateSlug && ! isSpecificSupportedTypeSet
-				? [ __( 'Page' ) ]
-				: BREADCRUMB_TYPES[ breadcrumbsType ].placeholderItems ),
-		].filter( Boolean );
+		const placeholderItems = [];
+		if ( showHomeLink ) {
+			placeholderItems.push( __( 'Home' ) );
+		}
+		if ( templateSlug && ! postId ) {
+			placeholderItems.push( __( 'Page' ) );
+		} else {
+			placeholderItems.push(
+				_showTerms ? __( 'Category' ) : __( 'Ancestor' ),
+				__( 'Parent' )
```

Comment:
This is a change in behaviour, as Parent will appear even when `_showTerms`. Intentional?

Reply:
Oh... That was not intentional. Fixed.